R/get_statistic.R

```diff
@@ -972,6 +972,47 @@ get_statistic.mlogit <- function(x, ...) {
   }
 }
 
+#' @export
+get_statistic.mclogit <- function(x, ...) {
+  if (requireNamespace("mclogit", quietly = TRUE)) {
```

Comment:
Why not use `insight::check_if_installed()` here to avoid control flow?

Reply:
was too lazy, just copy/pasted ;-)